MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -185,6 +194,34 @@ - (MSALPublicClientApplication *)msalTestPublicClientApplication
     return application;
 }
 
+- (MSALInteractiveTokenParameters *)tokenParams:(BOOL)isSSOSeedingCall
+{
+    MSALTestAppSettings *settings = [MSALTestAppSettings settings];
+    NSMutableArray<NSString *> *ssoSeedingScopes = [NSMutableArray new];
+    [ssoSeedingScopes addObject:@"urn:ms-drs:enterpriseregistration.windows.net/.default"];
```

Comment:
this will be different for other clouds

Reply:
Address, will update these code to get the value from WPJ first.